routes/ama.js

```diff
@@ -14,6 +16,16 @@ var queries = {
     update: "UPDATE ama_questions SET answer_text = "
 };
 
+let transporter = nodemailer.createTransport({
```

Comment:
I just tried this and it broke things. Maybe it's because I don't know how node handles it?